WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -53,6 +53,7 @@ + (NSArray *)routes
       [NSString stringWithFormat:@"%@ is not a valid URL", url]
     );
   }
+  #pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
1. You should use `#pragma clang diagnostic push` and `#pragma clang diagnostic pop` to make it local.
2. I agree with @mykola-mokhnach. You should add `if` that checks for the new API and use old API in case it is not available. Compiler SHOULD be smart enough to not generate warning